tsconfig.json

```diff
@@ -13,7 +13,6 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "isolatedModules": true,
```

Comment:
This change is going to be reverted every time the local server starts. Not allowed,

Reply:
Reverted